tools/init-tag-data/agency.yml

```diff
@@ -0,0 +1,36 @@
+- title: 'California Department of Aging'
+  acronym: CDA
+- title: 'California Department of Child Support Services'
+  acronym: DCSS
+- title: 'California Department of Social Services'
+  acronym: DSS
+- title: 'California Department of Community Services & Development'
+  acronym: CCSD
+- title: 'Department of Developmental Services'
+  acronym: DDS
+- title: 'California Emergency Medical Services Authority'
+  acronym: EMSA
+- title: 'Department of Health Care Services'
+  acronym: DHCS
+- title: 'Department of Managed Health Care'
+  acronym: DMHC
+- title: 'California Department of Public Health'
+  acronym: CDPH
+- title: 'Department of Rehabilitation'
+  acronym: DOR
+- title: 'Department of Social Services'
```

Comment:
We've got DSS & CDSS, both referring to (California) Department of Social Services. Should these be merged?

Reply:
Removing, thanks for the eyes